code-challenges.js

```diff
@@ -3,8 +3,21 @@
 
 // --------------------1) Create a function that returns the first 10 numbers of the Fibonacci sequence in an array. Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
-
-
+// function that takes in a certain number of indexes
+let fibonacci = (number) =>
+{
+// if the number inputed is 1 return [0,1]
+  if (number===1) {
+    return [0,1]
+  } else {
+// variable  that includes the fibonacci formula and pushes the answer into solution variable
+    let solution = fibonacci(number - 1)
+    solution.push(solution[solution.length - 1] + solution[solution.length - 2])
+    return solution
+  }
+}
+
+ console.log(fibonacci(9))
```

Comment:
Super awesome! You almost there! This is a REALLY hard challenge. 

I would encourage you to pay attention to our expected output. If we run this code, the output is `[0, 1,  1,  2,  3, 5, 8, 13, 21, 34]`

The expected output is `[1, 1, 2, 3, 5, 8, 13, 21, 34, 55]`

So this means, we're missing a 1 and the final 55. This is just due to your sequence starting with a 0, rather than a 1. 

Again, you did REALLY good on getting to your solution! I'd be super proud of this code :)